tasks/aliases.yaml

```diff
@@ -5,6 +5,7 @@ test:
   - connect:serve
   - sauce_connect
   - protractor
```

Comment:
It's no worry! I clearly didn't pay full attention when resolving conflicts in rebasing =D